README.md

```diff
@@ -46,11 +46,13 @@ The library is based on node.js which can be natively integrated with external C
 * [Firefox](https://www.mozilla.org/en-US/firefox/products/)
 
 **Server dependencies**: 
-* [YARP](https://github.com/robotology/yarp) (Duh!). With OpenCV!
+* [OpenCV](https://docs.opencv.org/trunk/d7/d9f/tutorial_linux_install.html)
+* [YARP](https://github.com/robotology/yarp) (Duh!). With OpenCV settings.
 * [node.js](https://nodejs.org/en/). Version >= 4.2.2. Follow the [official guide](https://nodejs.org/en/download/package-manager/). 
 
 **Note**: 
 - **Please make sure** that your Node version is >= 4.2.2
+- **Add OpenCV settings to YARP**: in your `yarp/build` directory rerun `ccmake ..`; set `CREATE_DEVICE_LIBRARY_MODULES ON`, then set `OpenCV_DIR` to your open cv directory (for example, /usr/local/share/OpenCV). Then re-run `sudo make install`. 
```

Comment:
`CREATE_DEVICE_LIBRARY_MODULES` is not concerned with OpenCV dependencies, unless there's something implicit here still unsaid.

However, all the info to deal with `OpenCV_DIR` along with pointers to cmake configuration and install process are really unnecessary and do make the instructions a bit more cluttered. My suggestion is therefore to remove this section 😉

Reply:
or maybe link to a dedicated page on how to use OpenCV with yarp?